gflow/config.py

```diff
@@ -0,0 +1,130 @@
+import getopt
+import ConfigParser
+import sys
+import logging
+import os
+
+
+class Config(object):
+
+    def __init__(self, configfile=None):
+
+        self.logger = logging.getLogger('gflow.Config')
+
+        self.logger.info("Reading config file")
+        if not configfile:
+            configfile = self.parse_options()
+
+        config = ConfigParser.RawConfigParser()
+        config.read(configfile)
+
+        try:
+
+            self.logger.debug("Storing Galaxy credentials")
+            self.galaxy_url = os.environ.get("GALAXY_URL", None)
```

Comment:
I decided to switch to environment variables because the user could just set them once and then all the tests would run and any config files they create will also use the same credentials.